### PR TITLE
fetch docs

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -251,7 +253,7 @@ markdown_extensions:
 plugins:
   - search
   # - mkdocs-versioning:
-  #     version: 1.0
+  #     version: 1.0version
```
Comment (whitewum) [inline mkdocs.yml:256]: why?

File: mkdocs.yml [hunk 2]
```diff
@@ -148,15 +148,17 @@ nav:
 
       - Vertex statements:
           - INSERT VERTEX: 3.ngql-guide/12.vertex-statements/1.insert-vertex.md
+          - DELETE VERTEX: 3.ngql-guide/12.vertex-statements/4.delete-vertex.md
```
Comment (whitewum) [inline mkdocs.yml:151]: why 4

File: docs-2.0/3.ngql-guide/13.edge-statements/5.fetch-edge.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# FETCH EDGE
+
+```ngql
+FETCH PROP ON <edge_type> <vid> -> <vid>[@<rank>] [, <vid> -> <vid> ...] [YIELD [DISTINCT] <return_list>]
+
+return_list ::=
+    [col_name [AS col_alias] [, col_name [AS col_alias]] ...]
+```
+
+Use `FETCH` to retrieve properties for edges. [Pipe](../5.operators/4.pipe.md) and user-defined variables (doc TODO) are permitted in a `FETCH` statement.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/13.edge-statements/5.fetch-edge.md:10]: add two examples

File: docs-2.0/3.ngql-guide/12.vertex-statements/5.fetch-vertex.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+vertex_id_list ::=
+  [vertex_id [, vertex_id] ...]
+
+return_list ::=
+    [col_name [AS col_alias] [, col_name [AS col_alias]] ...]
+```
+
+Use `FETCH` to retrieve properties for vertices. [Pipe](../5.operators/4.pipe.md) and user-defined variables (doc TODO) are permitted in a `FETCH` statement.
+
+## Example
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/12.vertex-statements/5.fetch-vertex.md:18]: two examples

File: mkdocs.yml (hunk 1 above)
Comment (Amber1990Zhang) [reply mkdocs.yml:256]: I remove this code. It's not in use.

File: docs-2.0/3.ngql-guide/12.vertex-statements/5.fetch-vertex.md [hunk 5]
```diff
@@ -15,4 +15,59 @@ return_list ::=
...
 ## Example
+
+```nql
+nebula> FETCH PROP ON player "player101";
++-------------+---------------+------------+
+| VertexID    | player.name   | player.age |
++-------------+---------------+------------+
+| "player101" | "Tony Parker" | 36         |
++-------------+---------------+------------+
+
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/12.vertex-statements/5.fetch-vertex.md:32]: add one line:
// two tags: team and player

File: docs-2.0/3.ngql-guide/13.edge-statements/5.fetch-edge.md [hunk 6]
```diff
@@ -8,3 +8,58 @@ return_list ::=
 ```
 
 Use `FETCH` to retrieve properties for edges. [Pipe](../5.operators/4.pipe.md) and user-defined variables (doc TODO) are permitted in a `FETCH` statement.
+
+- Only one edge type is permitted in a singe `FETCH` statement.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/13.edge-statements/5.fetch-edge.md:12]: ** ONLY ONE** edge type

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**